content/ros/concepts.md

```diff
@@ -4,5 +4,7 @@ Responsible: Kiran Vasudev | Pranav
 
 Content:
 * Why ROS?
-* Introduction to topics, nodes, messages...
+* Introduction to topics, nodes, messages,rqt graph...
+* Basic commands related to the concepts discussed
+* Explanation with turtlesim
```

Comment:
As long as the demo focuses on the concepts, e.g. showing how many nodes are running, which topics are there showing the node/topic graph, this is ok.

Reply:
You mean the presentation and not a programmatic demo right?

Reply:
Yes, I mean that it is alright to show the turtlesim to explain the concepts, showing for example which nodes are running and which topics are available. The focus should be on the concepts and not in commands such as `rostopic list` or `rosnode list`. Using `rqt_graph` would be better to visually show the nodes and topics.